src/common/xo/index.js

```diff
@@ -1428,6 +1428,15 @@ export const deleteMessage = message => (
   _call('message.delete', { id: resolveId(message) })
 )
 
+export const deleteMessages = logs => (
+  confirm({
+    title: _('deleteLogsConfirm'),
+    body: _('deleteLogsConfirmMessage')
+  }).then(
+    () => Promise.all(map(logs, deleteMessage))
```

Comment:
Add `noop` as second parameter otherwise there will be an uncaught error when the user closes the modal.